server/tsconfig.json

```diff
@@ -1,3 +1,6 @@
 {
-  "extends": "../tsconfig.build.json"
+  "extends": "../tsconfig.build.json",
+  "compilerOptions": {
+    "outDir": "./build"
```

Comment:
Wouldn't it make sense to add this to the `tsconfig.build.json` as I think we would want the `build` folder to exist for every package that could be built. (We wouldn't want to use dist somewhere and build somewhere else).

Reply:
Adding this line to the `tsconfig.build.json` resulted in a build without the src directory:
<img width="407" alt="image" src="https://user-images.githubusercontent.com/60812202/189077080-3c5c1b76-a452-483a-b5e8-07985811a896.png">

Adding it to the `tsconfig.json` in the `server` directory resulted in a correct build. 

Do you have any idea how this happens?

Reply:
No, not really. We can leave it as is.